components/general/_generalpreview.hbs

```diff
@@ -6,7 +6,7 @@
     <title>{{ title }}</title>
 </head>
 <body>
-    {{{ inline "components/general/icons/icon-sprite.svg" }}}
+    {{{ inline "public/images/icons-sprite.svg" }}}
     {{{ yield }}}
 </body>
 </html>
```

Comment:
but if I remove this file we don't have one main preview and it looking bad when you start fractal and go into the general. The best solution will create main preview file (copy this from `block` dir)